### ISSUE TITLE
Typo in ComputeMSE_ (DirectXTexMisc.cpp)

### ISSUE THREAD

**Issue by MLPMil** [Microsoft/DirectXTex#54]
Hello,
I was opening this issue when you did last commit...
However, at https://github.com/Microsoft/DirectXTex/blob/31b6c9309c489ed90a012a06d2915f2b4c256d25/DirectXTex/DirectXTexMisc.cpp#L129
I think that the code

```
if (flags & CMSE_IMAGE2_X2_BIAS)  
{  
	v1 = XMVectorMultiplyAdd(v2, two, g_XMNegativeOne);  
}  
```

should be

```
if (flags & CMSE_IMAGE2_X2_BIAS)
{
	v2 = XMVectorMultiplyAdd(v2, two, g_XMNegativeOne);
}
```

It's probably a "copy-and-paste" issue...

Regards
mlpmil

**Comment by walbourn** [comment]
Thanks! Great spot!

**Comment by walbourn** [comment]
Fixed in commit [52aead71d7c086f4750ab206d9be8846e84085b3](https://github.com/Microsoft/DirectXTex/commit/52aead71d7c086f4750ab206d9be8846e84085b3)

**Comment by walbourn** [comment]
Fix included in the July 2017 release
